src/Controls/src/Core/Application.cs

```diff
@@ -164,11 +166,14 @@ public AppTheme UserAppTheme
 			set
 			{
 				_userAppTheme = value;
-				TriggerThemeChangedActual(new AppThemeChangedEventArgs(value));
+				TriggerThemeChangedActual();
```

Comment:
This was actually incorrect and my test detected it. If you set the user theme to unspecified, then the event would say it was unspecified - but this is not a valid state for the theme as it falls back to the OS theme.